responder/jira/jira.go

```diff
@@ -0,0 +1,124 @@
+package jira
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/flanksource/commons/logger"
+	"github.com/flanksource/incident-commander/api"
+
+	jira "github.com/andygrunwald/go-jira"
+)
+
+type JiraOptions struct {
+	ProjectKey  string
+	Summary     string
+	Description string
+	IssueType   string
+}
+
+func GetClient(email, apiToken, url string) (*jira.Client, error) {
+
+	tp := jira.BasicAuthTransport{
+		Username: email,
+		Password: apiToken,
+	}
+
+	client, err := jira.NewClient(tp.Client(), url)
+	if err != nil {
+		return nil, err
+	}
+	return client, nil
+}
+
+func CreateIssue(client *jira.Client, opts JiraOptions) error {
+
+	i := jira.Issue{
+		Fields: &jira.IssueFields{
+			//Reporter: &jira.User{
+			//Name: opts.Reporter,
+			//},
+			Description: opts.Description,
+			Type: jira.IssueType{
+				Name: opts.IssueType,
+			},
+			Project: jira.Project{
+				Key: opts.ProjectKey,
+			},
+			Summary: opts.Summary,
+		},
+	}
+
+	issue, _, err := client.Issue.Create(&i)
+	if err != nil {
+		return err
+	}
+	logger.Debugf("[Jira] Issue created with ID: [%s]", issue.ID)
+
+	return nil
+}
+
+func AddComment(client *jira.Client, issueID, comment string) error {
+
```

Comment:
gofmt the file, there shouldn't be newlines at the beginning or end of a method

Reply:
I did gofmt it, it allows new lines, I'll remove them manually